src/AppRouter.js

```diff
@@ -17,37 +20,34 @@ const AppRouter = () => (
       path="/:topic"
       render={({ history, match }) => {
         if (topicKeys.includes(match.params.topic)) {
+          const activeTopic = topics.find(t => t.key === match.params.topic);
+          const { elements } = activeTopic;
+          let menuChildren = null;
+
+          switch (match.params.topic) {
+            case 'ch.sbb.handicap': {
+              menuChildren = <FeatureMenu popupComponents={POPUP_CONF} />;
+              break;
+            }
+            case 'ch.sbb.netzkarte':
+              menuChildren = <TrackerMenu />;
+              break;
+            default:
+              menuChildren = null;
+          }
+
           return (
             <TrafimageMaps
               history={history}
               activeTopicKey={match.params.topic}
               topics={topics}
               apiKey="5cc87b12d7c5370001c1d6551c1d597442444f8f8adc27fefe2f6b93"
-              elements={{
-                header: true,
-                footer: true,
-                menu: true,
-                permalink: true,
-                mapControls: true,
-                popup: true,
-                baseLayerToggler: true,
-              }}
+              elements={elements}
               initialState={{ ...match.params }}
               popupComponents={POPUP_CONF}
-              menuComponents={[
-                { component: ShareMenu, standalone: false },
-                {
-                  component: TrackerMenu,
-                  standalone: true,
-                  topic: 'ch.sbb.netzkarte',
-                },
-                {
-                  component: HandicapMenu,
-                  standalone: true,
-                  topic: 'ch.sbb.handicap',
-                },
-              ]}
-            />
+            >
+              <Menu subMenus=<ShareMenu />>{menuChildren}</Menu>
```

Comment:
This definitinon is a bit confusing, couldn't we have something like :
```
<Menu>
    <TopicsMenu>
        <ShareMenu/>
    </TopicsMenu>
    {menuChildren}
</Menu>
```
Or if `ShareMenu` is always there we can get rid of `subMenus` property, and put `ShareMenu` directly in `Menu`

Reply:
If this can be done easily, please fix this for me, @edellucien.
Otherwise we leave it like this for now. OK?

Reply:
ok